atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/priority/NextTableToSweepProvider.java

```diff
@@ -100,10 +135,11 @@ private TableReference getRandomValueFromList(List<TableReference> tables) {
                 ? LoggingArgs.safeTableOrPlaceholder(chosenTable.get()).toString()
                 : "no table";
 
-        log.debug("Chose {} from scores: {}, unsafeScores: {}",
+        log.debug("Chose {} from scores: {}, unsafeScores: {}, overrides: {}",
                 SafeArg.of("chosenTable", chosenTableString),
                 SafeArg.of("scores", safeTableNamesToScore),
-                UnsafeArg.of("unsafeScores", scores));
+                UnsafeArg.of("unsafeScores", scores),
+                SafeArg.of("overrides", sweepPriorityOverrideConfig));
```

Comment:
The overrides contains table names so we should treat it as unsafe, or sanitize it as we do above with `safeTableNamesToScore `.  The set of people able to view the safe logs may not be the same as the set of people able to view and edit the config (and hence know the table names).

Reply:
Fair enough - I marked it as Safe based on assumptions on our current security model, but 'config view' and 'safe-log view' are indeed arguably different permissions. Will change to Unsafe for now - sanitising is not incredibly hard, but looks to get quite messy.